### PR TITLE
Added SolutionInfo as central versioning point

### PR DESCRIPTION
**Pull request by dejangeci**
Removed extra stuff from AssemblyInfos and added SolutionInfo as central versioning point so there is no need to update each project separately

### CODE REVIEW

File: src/Audio/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,33 +1,8 @@
...
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("SFML")]
-[assembly: AssemblyCopyright("Copyright © Laurent Gomila")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
 
-// L'affectation de la valeur false à ComVisible rend les types invisibles dans cet assembly 
-// aux composants COM. Si vous devez accéder à un type dans cet assembly à partir de 
-// COM, affectez la valeur true à l'attribut ComVisible sur ce type.
 [assembly: ComVisible(false)]
```
Comment (zsbzsb) [inline src/Audio/Properties/AssemblyInfo.cs]: Not a big issue, but while you are changing this, would you stuff mind to move the com visible attribute also and set it to true? (all others except audio are visible)

**Comment by dejangeci** [comment]
Only reason why I didn't move it was because Audio had ComVisible(false). I figured @LaurentGomila  had a  specific reason for this, but it was probably auto-generated like that.

**Comment by dejangeci** [comment]
Done.

**Comment by zsbzsb** [comment]
Tested and works great, thanks!

**Pull request merged**